runtime/tools/spec-builder/test/actual-raw-files/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Actually, we do not even need this `.gitignore` file because we are clearing the directory altogether after the tests.